src/logic/community/communities.ts

```diff
@@ -59,7 +68,7 @@ export async function createCommunityComponent(
         }
       }
 
-      return toCommunityWithMembersCount(community, membersCount)
+      return toCommunityWithMembersCount(community, membersCount, voiceChatStatus)
```

Comment:
Do we need to extend this function? could we just extend the type to include `voiceChatStatus` properties and handle it like we did the `ownerName`? https://github.com/decentraland/social-service-ea/pull/191/files#diff-4eed535b564dded7d489f223b66aedc5356782816d9838d5ca9874b100e5f1c0R74